README.md

```diff
@@ -55,23 +55,26 @@ cp .env.example .env
 Install dependencies
 
 ```
-pip3 install -r requirements.txt
+pip3 install pipenv
+pipenv --python 3.7
+pipenv install
+pipenv shell
 ```
 
 Run Database migrations
 
 ```
-python3 manage.py migrate
+python manage.py migrate
```

Comment:
This command would fail in the systems where default python is python2 because of the syntax in `manage.py`

Reply:
pipenv shell changes python to default to python3.